python/raydp/torch/config.py

```diff
@@ -16,7 +16,15 @@ class TorchConfig(RayTorchConfig):
     def backend_cls(self):
         return EnableCCLBackend
 
-def ccl_import():
+def libs_import():
+    """try to import IPEX and oneCCL.
+    """
+    try:
+        import intel_extension_for_pytorch
+    except ImportError:
+        raise ImportError(
+            "Please install IPEX"
```

Comment:
can you rewrite this to "Please install intel_extension_for_pytorch"?

Reply:
done